### DISCUSSION TITLE
Obs Studio Widget

### DISCUSSION THREAD

**Discussion by mchamsteryt2** [baffler/Transparent-Twitch-Chat-Overlay#109] [Q&A]
How Do Make This Also Be An Obs Studio Widget Because The Last Time I Tried It Didn't Work

**Comment by AgentZman** [comment]
While you have the "baffler" open for whatever purpose such as twitch chat or some other url overlay you want on your game screen. Or even minimized if you like. THEN you open OBS and add Window Capture. Then from the list you select whatever window you want. IN this case it will be the "TwitchChatOverlay window. This way you have in game overlay and also OBS overlay as you like. 
Hope this helps. works great.
